<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tabs</class>
 <widget class="QWidget" name="Tabs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 lightgreen, stop:1 darkgreen);
}

</string>
  </property>
  <widget class="QPushButton" name="TaskManager">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>81</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Task Manager&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Task Manager</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HealthVitalStats">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Health Vital Stats</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DietPlan">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>200</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Diet Plan Management</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FitnessRoutine">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Fitness Routine</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MentalHealth">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Mental Health</string>
   </property>
  </widget>
  <widget class="QLabel" name="Tabs_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; font-style:italic;&quot;&gt;Matrix Tabs&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BacktoSign">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Sign-In</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Close">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
